client/question/domain/FeedbackObjectFactory.js

```diff
@@ -177,6 +185,24 @@ tie.factory('FeedbackObjectFactory', [
       return this._feedbackCategory;
     };
 
+    /**
+     * A getter for the _errorLineNumber property.
+     *
+     * @returns {string}
+     */
+    Feedback.prototype.getErrorLineNumber = function() {
+      return this._errorLineNumber;
+    };
+
+    /**
+     * A setter for the _errorLineNumber property.
+     *
+     * @param {integer} errorLine to set the _errorLineNumber property to.
```

Comment:
Param name in docstring does not match arg name

Reply:
Fixed.